src/library_browser.js

```diff
@@ -1381,6 +1400,13 @@ var LibraryBrowser = {
     styleSheet.insertRule('canvas.emscripten { border: 1px solid black; cursor: none; }', 0);
   },
 
+  emscripten_set_element_style__proxy: 'sync',
+  emscripten_set_element_style__sig: 'viii',
+  emscripten_set_element_style: function(selector, property, value) {
+    var element = Module[selector];
+    if (element) element.style[property] = UTF8ToString(value);
+  },
```

Comment:
I'm unclear on the benefit of this function. It looks on things on `Module` - what elements are expected to be there? Is it just the canvas?

Reply:
Also, if it's just the canvas, we already have `emscripten_set_canvas_element_size` which looks related?

Reply:
> Also, if it's just the canvas, we already have `emscripten_set_canvas_element_size` which looks related?

Yeah, it could have a better name, perhaps `emscripten_set_canvas_element_style`? The purpose is to allow SDL2 to set the cursor style in a way that works with `PROXY_TO_PTHREAD`. You can see the intended usage here: https://github.com/emscripten-ports/SDL2/pull/77/commits/f0426362f751aabbda4f3d5e561e445566db4005.

Reply:
Thanks for the link! Now I see the issue here, I think.

For something like this, perhaps we can use `MAIN_THREAD_EM_ASM`? I see that isn't documented well, I'll look into that. But basically it lets you proxy a call to the main thread and run some EM_ASM code there.

Reply:
That will work, I can remove this API, and update the SDL2 branch to use this syntax.